src/software/new_geom/convex_polygon.cpp

```diff
@@ -50,25 +50,25 @@ bool ConvexPolygon::isConvex()
 
         // Calculate sign flips using the next edge vector ("next_to_curr"),
         // recording the first sign
-        if (next_to_curr.x() > GeomConstants::FIXED_EPSILON)
+        if (next_to_curr.x() > FIXED_EPSILON)
         {
             if (x_sign == 0)
             {
                 x_first_sign = 1;
             }
-            else if (x_sign < -GeomConstants::FIXED_EPSILON)
+            else if (x_sign < -::FIXED_EPSILON)
```

Comment:
some of these have `::` and some don't is that intentional?

Reply:
ah just saw Jons comment, still confused as to why there are two? Do they both resolve to the same scope?

Reply:
ahhh just saw Matts comment, we should remove the `::` in this file since they are the same

Reply:
I missed this

Reply:
Whoops I also missed this with find/replace. Fixed all instances :tm: of this now.